### PR TITLE
Add Donetick

### PR DESCRIPTION
**Pull request by meauxt**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/donetick.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
```
Comment (Rabenherz112) [inline software/donetick.yml:4]: Please make the description under 250 characters long.
```suggestion
description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
```

File: software/donetick.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Nodejs
+  - Docker
```
Comment (Rabenherz112) [inline software/donetick.yml:10]: Nodejs is not needed for installation, therefore can be excluded
```suggestion
platforms:
  - Go
  - Docker
```

File: software/donetick.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - Task Management & To-do Lists
+  - Automation
+  - Personal Dashboards
+
```
Comment (Rabenherz112) [inline software/donetick.yml:15]:
```suggestion
tags:
  - Task Management & To-do Lists
```

File: software/donetick.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+depends_3rdparty: true
```
Comment (Rabenherz112) [inline software/donetick.yml:16]: Where does it depend on a 3rd party? I couldn't find it when I quickly looked over the repo.
```suggestion
depends_3rdparty: true
```

File: software/donetick.yml (hunk 1 above)
Comment (meauxt) [reply software/donetick.yml:4]: Hi @Rabenherz112 ! I try a few character counters and see this as 207 characters. also, the suggested change is exactly the same

File: software/donetick.yml (hunk 4 above)
Comment (meauxt) [reply software/donetick.yml:12]: I was not sure if 3rd_party here mean like not having zero dependency on other libraries :) i clearly miss read it

File: software/donetick.yml (hunk 1 above)
Comment (Rabenherz112) [reply software/donetick.yml:4]: Nevermind, I accidentally copied your description 2x in the word counter.

File: software/donetick.yml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Docker
+tags:
+  - Task Management & To-do Lists
+depends_3rdparty: false
```
Comment (Rabenherz112) [inline software/donetick.yml:12]:
```suggestion
```
This can be excluded completely, as it by default false

Comment (meauxt) [reply software/donetick.yml]: sounds good! omitted!

File: software/donetick.yml (hunk 4 above)
Comment (meauxt) [reply software/donetick.yml]: oh got it! Thank you @Rabenherz112

File: software/donetick.yml (hunk 1 above)
Comment (meauxt) [reply software/donetick.yml:4]: That's cheating :)

**Review by Rabenherz112 — APPROVED** [review]

File: software/donetick.yml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+name: "Donetick"
+website_url: "https://donetick.com"
+source_code_url: "https://github.com/donetick/donetick"
+description: "Task and chore management tool for personal and family use. with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API. with simple and modern design."
```
Comment (nodiscc) [inline software/donetick.yml:4]:
```suggestion
description: "Task and chore management tool for personal and family use, with advanced scheduling, flexible assignment, and group sharing capabilities, detailed history, automation via API, simple and modern design."
```

**Review by nodiscc — APPROVED** [review]

**Pull request merged**